backend/app.py

```diff
@@ -63,20 +63,37 @@ async def login(key: str | None = None):
 
 
 @game_router.get(
-    "/current_state",
+    "/current_state/{game_id}",
     tags=["voting"],
 )
-async def get_current_state():
-    return Response(status_code=HTTPStatus.NO_CONTENT)
+async def get_current_state(game_id: int):
+    with Session(engine) as session:
+        game = session.exec(select(Game).where(Game.id == game_id)).first()
+        if not game:
+            return Response(status_code=HTTPStatus.BAD_REQUEST)
+        
+        return {"game_id": game.id, "state": game.state, "current_voting_event":game.current_voting_event, "current_voting_event": game.current_voting_event}
```

Comment:
Duplicate current_voting_event here. Also, the Game model does not have a current_voting_event attribute, so this would cause an error.